### PR TITLE
[Remove Blocking usages]LPS-187720 portal-kernel | AssetCategoryConfigurationProviderUtil

### PR DESCRIPTION
**Pull request by JanisZhang**
https://issues.liferay.com/browse/LPS-187720

### CODE REVIEW

**Comment by JanisZhang** [comment]
ci:test:startup

File: portal-kernel/src/com/liferay/asset/kernel/configuration/provider/AssetCategoryConfigurationProviderUtil.java [hunk 1]
```diff
@@ -47,6 +47,6 @@ public static boolean isSearchHierarchical(long companyId) {
 			ServiceProxyFactory.newServiceTrackedInstance(
 				AssetCategoryConfigurationProvider.class,
 				AssetCategoryConfigurationProviderUtil.class,
-				"_assetCategoryConfigurationProvider", true);
+				"_assetCategoryConfigurationProvider", false);
```
Comment (dantewang) [inline portal-kernel/src/com/liferay/asset/kernel/configuration/provider/AssetCategoryConfigurationProviderUtil.java:50]: @JanisZhang Please modify the commit message, just say what you did, if there's no specific reason but we think it's safe to do so.

**Review by dantewang — CHANGES_REQUESTED** [review]

File: portal-kernel/src/com/liferay/asset/kernel/configuration/provider/AssetCategoryConfigurationProviderUtil.java (hunk 1 above)
Comment (JanisZhang) [reply portal-kernel/src/com/liferay/asset/kernel/configuration/provider/AssetCategoryConfigurationProviderUtil.java:50]: @dantewang force-pushed, thanks!

**Comment by dantewang** [comment]
ci:test:sf

**Comment by dantewang** [comment]
ci:test:stable

**Comment by dantewang** [comment]
ci:forward:force

cc @shuyangzhou @tinatian

**Comment by dantewang** [comment]
ci:stop

**Comment by dantewang** [comment]
@JanisZhang a message as simple as "LPS-187720 Set ServiceProxyFactory blocking parameter to false in XXX" should be enough, if you don't have anything to say from reviewing git history.
cc @W2jx @LisztLiszt @slnn

**Comment by JanisZhang** [comment]
> @JanisZhang a message as simple as "LPS-187720 Set ServiceProxyFactory blocking parameter to false in XXX" should be enough, if you don't have anything to say from reviewing git history. cc @W2jx @LisztLiszt @slnn

Gotcha, thanks!

**Comment by JanisZhang** [comment]
Force-pushed. I only changed the commit message. It shouldn't affect the test results. Thanks!

**Comment by dantewang** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/136895

**Pull request closed without merging**